src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -107,6 +110,24 @@ public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpReq
             }
         }
 
+        [ConditionalTheory(nameof(IsWindows10Version1607)), MemberData(nameof(Http2Servers))]
```

Comment:
We already have a test like this in System.Net.Http.Functional.Tests, right?  Do you want to enable that one on appropriate versions of Windows as well?